### PR TITLE
SPCTR-3534: PHPStan baseline part 3

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
PHPStan baseline clear

### Types of changes

- includes/blocks/post-grid/frontend.css.php
- includes/blocks/post-grid/attributes.php
- includes/blocks/post-carousel/frontend.css.php
- includes/blocks/post-carousel/attributes.php
- includes/blocks/modal/frontend.js.php
- includes/blocks/modal/frontend.css.php
- includes/blocks/modal/attributes.php
- includes/blocks/marketing-button/frontend.css.php
- includes/blocks/marketing-button/attributes.php

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/modal/frontend.css.php [hunk 1]
```diff
@@ -3,7 +3,8 @@
  * Frontend CSS loading File.
  *
  * @since 2.2.0
- *
+ * @var string[] $attr
```
Comment (AbhijitNage123) [inline includes/blocks/modal/frontend.css.php:6]: @yash Do we need to put mixed[] every where. $attr is mixed type every time. Is there any reason here?

Comment (yash-tingre-10) [reply includes/blocks/modal/frontend.css.php:6]: It was going error when use mixed for for a function paramater only accepting string but mixed given so when I changed to string [] that issue was resolved

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/modal/attributes.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	'content'
 );
 
-$additional_attributes = apply_filters(
+$additional_attributes = (array) apply_filters(
```
Comment (adi3890) [inline includes/blocks/modal/attributes.php:18]: instead of type casting chek for datatype.
using **is_array** as most of the time it should be array

Comment (yash-tingre-10) [reply includes/blocks/modal/attributes.php]: Done with the changes

**Pull request merged**